test_ngtf.py

```diff
@@ -24,223 +24,8 @@
 import platform
 from distutils.sysconfig import get_python_lib
 
-from tools.build_utils import load_venv, command_executor
```

Comment:
Some files like this: `ngraph-tf/test/ci/macos/run-mac-build-test-ngtf.sh` depend on `./test_ngtf.sh`. May need to rewire accordingly

Reply:
Yep - in a future PR.